stdlib/2/urllib2.pyi

```diff
@@ -11,8 +13,8 @@ class URLError(IOError):
 
 class HTTPError(URLError, addinfourl):
     code = ...  # type: int
-    headers = ...  # type: Dict[str, str]
-    def __init__(self, url, code, msg, hdrs, fp) -> None: ...
+    headers = ...  # type: Message
```

Comment:
Nit: For new and changed code we prefer variable annotation syntax: `headers: Message`. (Or rather `header: Mapping[str, str]` in this case.)

Reply:
Fixed by fc1ef1bcacbeebff596a994a8831e6df80bdb07c and fc2feed713938f1a74b59917db25c792f71ad02f